app/pathogen/arbovirus/dashboard/(map)/MapArbovirusFilter.tsx

```diff
@@ -2,17 +2,24 @@ import { Checkbox } from "@/components/ui/checkbox";
 import { pathogenColorsTailwind } from "./ArbovirusMap";
 import { Button } from "@/components/ui/button";
 import SectionHeader from "@/components/customs/SectionHeader";
-import { PathogenContextType, PathogenContextActionType } from "@/contexts/pathogen-context/pathogen-context";
-import { ArbovirusEstimate } from "@/contexts/pathogen-context/pathogen-contexts/arbo-context";
+import { PathogenContextType } from "@/contexts/pathogen-context/pathogen-context";
+import { cn } from "@/lib/utils";
+import { SendFilterChangeDispatch } from "@/components/customs/filters";
+import { FilterableField } from "@/components/customs/filters/available-filters";
 
-interface MapArbovirusFilterProps {
+interface MapArbovirusFilterProps<TEstimate extends Record<string, unknown>> {
```

Comment:
Generic here and in the other places because this component should accommodate SC2 estimates as well as arbo ones.

This component in particular is pretty arbo specific and there's definitely some cleanup to be done with this component in particular but just explaining the idea.